shared/uavobjectdefinition/inssettings.xml

```diff
@@ -18,6 +18,6 @@
 		<access gcs="readwrite" flight="readwrite"/>
 		<telemetrygcs acked="true" updatemode="onchange" period="0"/>
 		<telemetryflight acked="true" updatemode="onchange" period="0"/>
-		<logging updatemode="manual" period="0"/>
+		<logging updatemode="manual" period="1"/>

```

Comment:
I'd rather go for 65535, if only because the resolution is sufficiently limited that we shouldn't sacrifice over 32 seconds of resolution in order to get a single negative value.